tests/extension.spec.ts

```diff
@@ -94,19 +113,13 @@ describe("Extension unit tests", () => {
       configSettingsMock.verify();
     });
 
-    it("activate - add subscriptions", async () => {
```

Comment:
We dont need this test or is it tested in other place?

Reply:
I check it here https://github.com/SAP/vscode-mta-tools/blob/move_task_provider/tests/extension.spec.ts#L79